src/components/NcListItem/NcListItem.vue

```diff
@@ -345,7 +345,7 @@
 					@focus="showActions"
 					@focusout="handleBlur"
 					@click="onClick($event, navigate, routerLinkHref)"
-					@keydown.enter="onClick($event, navigate, routerLinkHref)"
+					@keydown.enter.prevent="$event.target.click()"
```

Comment:
Makes sense, but programmatically clicking on a link is blocked by some browsers.

Reply:
Like?

Reply:
Enter (and click) should work natively for non-router links

Reply:
> Like?

Firefox

Reply:
> Firefox

Are you sure? This works fine for me on ff: https://codepen.io/pulsejet/pen/MWRKKoK?editors=1111

Reply:
> > Like?
> 
> Firefox

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/click#browser_compatibility

Reply:
> Are you sure? This works fine for me on ff: https://codepen.io/pulsejet/pen/MWRKKoK?editors=1111

Sorry for the inaccuracy. Not click itself, but opening a page (pop-up) is blocked.

In your example, there is no actual link — it doesn't have non-empty `href` and doesn't open any page. Add href, `target="_blank"`, remove `event.preventDefault()`. It is supposed to open a page by Enter like on click, but it won't.

Reply:
Okay, then we throw out that handler. Just a conlick should be fine
https://codepen.io/pulsejet/pen/KKYVMob?editors=1111

Reply:
> target="_blank"

Just a note, I don't believe this is working right now anyway since it sets location directly.

Reply:
> Just a note, I don't believe this is working right now anyway since it sets location directly.

For me, it doesn't work completely correctly. It makes both - navigation on the current page (via window.location) and opening a new tab.

Reply:
Huh? Where the window.location (after this patch)

Reply:
I mean, in `master`. In this PR, everything seems to work in every case.